Gemfile

```diff
@@ -32,7 +32,7 @@ gem 'aws-sdk-sns'
 
 gem 'laa-criminal-legal-aid-schemas',
     github: 'ministryofjustice/laa-criminal-legal-aid-schemas',
-    tag: 'v1.7.1'
+    branch: 'CRIMAPP-1964-maat-language'
```

Comment:
Will be updated with released version once https://github.com/ministryofjustice/laa-criminal-legal-aid-schemas/pull/236 released

Reply:
Approved you may update